anisoap/representations/ellipsoidal_density_projection.py

```diff
@@ -317,6 +317,36 @@ def contract_pairwise_feat(pair_ellip_feat, species):
 
     return ellip
 
+def normalize_basis(radial_basis: RadialBasis, features: TensorMap):
+    """
+    Multiply each value within each block of the features TensorMap by the appropriate normalization value.
+    These normalization values are given here: https://github.com/lab-cosmo/librascal/blob/a4ffbc772ad97ce6cbe9b46900660236b94d2ee2/bindings/rascal/utils/radial_basis.py#L100
+    This normalization scales down the GTO portion appropriately, but I'm still unsure what the normalizationr represents.
+    i.e. I'm not sure if the normalization ensures that the integral from 0 to inf = 1, or if the integral from 0 to inf
+    of the GTO^2 = 1, or something else.
+    
+    Parameters:
+        radial_basis: An instance of RaidalBasis
+        features: A TensorMap whose blocks' values we wish to normalize
+
+    Returns:
+        normalized_features: A copy of features with containing values multiplied by proper normalization factors
+    """
+    normalized_features = features.copy()
+    radial_basis_name = radial_basis.radial_basis
+    sigma = radial_basis.hypers["radial_gaussian_width"]
+    if radial_basis_name != "gto":
+        warnings.warn("Have not implemented normalization for non-gto basis, will return original values")
+        return features
+    for l, block in enumerate(normalized_features.blocks()):
+        for k, property in enumerate(block.properties):
+            n = property[0]
+            l_2n = l + 2 * n
+            N = np.sqrt(2 / (sigma ** (2 * l_2n + 3) * gamma(l_2n + 1.5)))
```

Comment:
I would make sure to explicitly import gamma from `scipy.special` so we have it explicitly in here and not just inherited through moment generator

Reply:
Done